### DISCUSSION TITLE
Migrate from Linux installation to Home Assistant addon

### DISCUSSION THREAD

**Discussion by Flappiee** [Koenkk/zigbee2mqtt#16967] [General]
How can i migrate from Linux Installation to Home Assistant Addon?

On my Linux installation i have a data folder that contains my database.db and configuration.yaml in /opt/zigbee2mqtt/data

In Home Assistant, after adding the repo and installing the addon, the configuration is in /usr/share/hassio/homeassistant/zigbee2mqtt

Is het simply copying the configuration.yaml and database.db to /usr/share/hassio/homeassistant/zigbee2mqtt ?

**Comment by Daniel15** [comment]
That *should* work, but I'm not familiar with how the Home Assistant addon works (I use Docker for everything). I think you should copy `state.json` across too though.
